app/templates/buyers/studios_start_page.html

```diff
@@ -67,11 +67,11 @@ <h2 class="govuk-heading-m">Evaluate</h2>
 
     <p class="govuk-body">Read more about <a class="govuk-link" href="https://www.gov.uk/guidance/how-to-hire-user-research-labs-on-the-digital-marketplace">how to hire user research labs</a>.</p>
```

Comment:
Is this `labs` being left as-is deliberately?

Reply:
Yes @bjgill that URL is on GOV.UK so not something we control directly (especially having no content designer currently).

Reply:
fixed link text in 7a96ca9, we cannot control the link URL as that is on GOV.UK.